lib/svg-preview-view.js

```diff
@@ -27,6 +27,17 @@ class SvgPreviewView extends View {
             this.text('transparent')
           })
         })
+        this.div({ class: 'image-controls-group btn-group' }, () => {
+          this.a({ class: 'btn', outlet: 'zoomOutButton' }, () => {
+            this.text('-')
+          })
+          this.a({ class: 'btn reset-zoom-button', outlet: 'resetZoomButton' }, () => {
+            this.text('100%')
```

Comment:
Done. I also added a check that the scale factor doesn't become zero or even negative.